src/mon/Elector.h

```diff
@@ -88,188 +91,42 @@ class Elector {
    */
   void cancel_timer();
 
-  /**
-   * Latest epoch we've seen.
-   *
-   * @remarks if its value is odd, we're electing; if it's even, then we're
-   *	      stable.
-   */
-  epoch_t epoch;
-
-  /**
-   * Indicates if we are participating in the quorum.
-   *
-   * @remarks By default, we are created as participating. We may stop
-   *	      participating if the Monitor explicitly calls
-   *	      Elector::stop_participating though. If that happens, it will
-   *	      have to call Elector::start_participating for us to resume
-   *	      participating in the quorum.
-   */
-  bool participating;
-
   // electing me
   /**
    * @defgroup Elector_h_electing_me_vars We are being elected
    * @{
    */
   /**
-   * Indicates if we are the ones being elected.
-   *
-   * We always attempt to be the one being elected if we are the ones starting
-   * the election. If we are not the ones that started it, we will only attempt
-   * to be elected if we think we might have a chance (i.e., the other guy's
-   * rank is lower than ours).
-   */
-  bool     electing_me;
-  /**
-   * Set containing all those that acked our proposal to become the Leader.
-   *
-   * If we are acked by everyone in the MonMap, we will declare
-   * victory.  Also note each peer's feature set.
-   */
-  map<int, elector_info_t> acked_me;
-  /**
-   * @}
-   */
-  /**
-   * @defgroup Elector_h_electing_them_vars We are electing another guy
-   * @{
-   */
-  /**
-   * Indicates who we have acked
+   * Map containing info of all those that acked our proposal to become the Leader.
+   * Note each peer's info.
    */
-  int	    leader_acked;
+  map<int, elector_info_t> peer_info;
```

Comment:
s/peer_info/peers/

Reply:
I like the extra specificity; it's not a messaging object or anything.